lib/FlashSale.js

```diff
@@ -70,6 +70,12 @@ class FlashSale {
      */
     this.isPopular = data.Popular;
 
+    /**
+     * Whether this item is show in the in-game market
+     * @type {boolean}
+     */
+    this.isShowMarket = data.ShowInMarket;
```

Comment:
if we're using it to represent whether or not it's shown in store, the conjugation of shown should be used, not show. `show` is grammatically confusing.

Reply:
`inMarket` is it true? or what is appropriate.

Reply:
![image](https://user-images.githubusercontent.com/7725575/55725078-87c29480-5a15-11e9-9f60-cc8b13681920.png)
yes

Reply:
`isShowMarket` is grammatically confusing in english, since, ideally, code should read like a sentence, like `is X show market` doesn't make sense to me. A more appropriate naming would be `isShownInMarket` or simply `shownInMarket`.